helm/blueapi/templates/init-configmap.yaml

```
@@ -1,7 +1,11 @@
+{{- if .Values.initContainer.enabled }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ include "blueapi.fullname" . }}-initconfig
 data:
-  initconfig.yaml: |-
-    {{- toYaml .Values.initContainer | nindent 4 }}
+  initconfig.yaml: 
+    scratch:
+      {{- toYaml .Values.worker.scratch | nindent 6 }}
+---
+{{- end }}
```

```suggestion
{{- end }}
---
```

nit: I think I'd prefer all of the yaml documents are self contained- if we pipe the output of helm into a file and it has a bunch of adjacent
```
---
---
---
``` 

then it looks a bit ugly but I'd rather have it than accidentally merge two documents